mmdeploy/apis/onnx/export.py

```diff
@@ -127,7 +127,14 @@ def wrapper(*arg, **kwargs):
             patched_model.forward = wrap_forward(patched_model.forward)
             patched_model.forward = partial(patched_model.forward,
                                             **input_metas)
-
+        # force to export on cpu
```

Comment:
What if here are op that only support gpu?

Reply:
maybe we could add a dummy custom op to make it runnable on cpu.